common/main/java/com/couchbase/lite/internal/core/C4Log.java

```diff
@@ -167,6 +167,8 @@ public boolean isInUse() {
         m.put(LogDomain.REPLICATOR, C4Constants.LogDomain.SYNC);
         m.put(LogDomain.QUERY, C4Constants.LogDomain.QUERY);
         m.put(LogDomain.LISTENER, C4Constants.LogDomain.LISTENER);
+        m.put(LogDomain.MULTIPEER, C4Constants.LogDomain.DISCOVERY);
+        m.put(LogDomain.PEER_DISCOVERY, C4Constants.LogDomain.P2P);
```

Comment:
Is this supposed to be a reverse mapping of the above?  If so it doesn't seem consistent:

Discovery -> Peer Discovery but Peer Discovery -> P2P for example

Reply:
Yes.  Good catch.